src/js/angular/similarity/controllers/similarity-list.controller.js

```diff
@@ -75,76 +47,17 @@ function SimilarityCtrl($scope, $interval, toastr, $repositories, $licenseServic
             });
     };
 
-    $scope.pullList = function () {
-        $scope.getSimilarityIndexes();
-        const timer = $interval(function () {
-            $scope.$broadcast('checkIsActive');
-            if ($('#indexes-table').attr('aria-expanded') !== 'false') {
-                $scope.getSimilarityIndexes();
-            }
-        }, 5000);
-        $scope.$on('$destroy', function () {
-            $interval.cancel(timer);
-        });
-    };
-
-    if ($scope.getActiveRepository()) {
-        if ($licenseService.isLicenseValid()) {
-            $scope.pullList();
-        }
-    }
-
-    let yasr;
-
-    $scope.$watch(function () {
-        return $repositories.getActiveRepository();
-    }, function () {
-        // Don't try to get namespaces for ontop or fedx repository
-        if ($scope.getActiveRepository() && !$scope.isActiveRepoOntopType() && !$scope.isActiveRepoFedXType()) {
-            $scope.getNamespacesPromise = RDF4JRepositoriesRestService.getNamespaces($scope.getActiveRepository())
-                .success(function (data) {
-                    checkAutocompleteStatus();
-                    $scope.usedPrefixes = {};
-                    data.results.bindings.forEach(function (e) {
-                        $scope.usedPrefixes[e.prefix.value] = e.namespace.value;
-                    });
-                    $scope.$on('$destroy', function () {
-                        if (yasr) {
-                            yasr.destroy();
-                        }
-                    });
-                    yasr = YASR(document.getElementById('yasr'), { // eslint-disable-line new-cap
-                        //this way, the URLs in the results are prettified using the defined prefixes
-                        getUsedPrefixes: $scope.usedPrefixes,
-                        persistency: false,
-                        hideHeader: true,
-                        pluginsOptions: YasrUtils.getYasrConfiguration()
-                    });
-                }).error(function (data) {
-                    toastr.error(getError(data), $translate.instant('get.namespaces.error.msg'));
-                });
-        }
-    });
-
-    function checkAutocompleteStatus() {
-        if ($licenseService.isLicenseValid()) {
-            $scope.getAutocompletePromise = AutocompleteRestService.checkAutocompleteStatus();
+    $scope.updateSimilarityIndexes = function () {
```

Comment:
You can just use arrow functions everywhere for consistency.

Reply:
Ok, I will do it.